.github/workflows/ci.yml

```diff
@@ -65,6 +65,12 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: "1.19"
+      - name: Install & Restart Kurtosis
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli
+          kurtosis engine restart
```

Comment:
Why does this require a restart after install?

Reply:
I have changed it to `start`. I am just super used to using `kurtosis engine restart` locally!